modules/gdscript/gdscript_editor.cpp

```diff
@@ -56,15 +56,15 @@ Ref<Script> GDScriptLanguage::get_template(const String &p_class_name, const Str
 
 	String _template = String() +
 					   "extends %BASE%\n\n" +
-					   "# class member variables go here, for example:\n" +
+					   "# Class member variables go here, for example:\n" +
 					   "# var a = 2\n" +
 					   "# var b = \"textvar\"\n\n" +
 					   "func _ready():\n" +
 					   "%TS%# Called when the node is added to the scene for the first time.\n" +
 					   "%TS%# Initialization here.\n" +
 					   "%TS%pass\n\n" +
 					   "#func _process(delta):\n" +
```

Comment:
What's the reasoning behind commenting out `process()` but not commenting `ready()`?

Reply:
Because if you declare `_process()` it'll imply `set_process(true)` which will technically decrease the performance since `_process()` will be called every frame (even if empty).

Reply:
That's only half an answer. I was leaning towards "why declare `ready()` if it's going to be empty?" 

From answering lots of beginner questions, it's very obvious that most new users start with the template as-is, so what's put in there has a huge influence on how they write their first scripts. *nobody* seems to understand that `pass` is a placeholder, and I've seen a bunch of misconceptions that `pass` is how you end a function.

What if it were like this:
```
extends Node

# Declare member variables here. Examples:
# var a = 2
# var b = "textvar"

# func _ready():
    # Code placed here runs when the node is added to the scene.

#func _process(delta):
#	# Called every frame. Delta is time since last frame.
#	# Update game logic here.
```

The commented `pass` is pointless, and just like `_process()`, `_ready()` can be uncommented if it's needed. If you're not going to uncomment, you don't need the `pass` anyway.

Reply:
Well, there was never much thought put into the wording itself. Someone added `_process()` because it's a common function but since that could have a performance impact it was just commented out.

Reply:
@cbscribe Prefer that, but would make it more verbose:
`# Code placed here runs once each time the node is added to a scene.`

Reply:
@mhilbrunner since Godot 3 `_ready()` runs only the first time the node enters the tree.